<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Evenement1</class>
 <widget class="QMainWindow" name="Evenement1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evenement</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>861</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 9pt &quot;Sylfaen&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>90</y>
        <width>451</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_44">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(155, 1, 1);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;STREAMFLIX&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_40">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(155, 1, 1);
image: url(:/new/prefix1/Logo.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>851</width>
        <height>551</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(155, 1, 1);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>321</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(155, 1, 1);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="title">
       <string>Fonctionnalité :</string>
      </property>
      <widget class="QLineEdit" name="rechercher">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rechercher_button_affichage">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>70</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 0);
border :1px solid rgb(255, 255, 255);
border-radius :10px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
</string>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pdf_button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>301</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 0);
border :1px solid rgb(255, 255, 255);
border-radius :10px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
</string>
       </property>
       <property name="text">
        <string>Générer un PDF</string>
       </property>
      </widget>
      <widget class="QComboBox" name="Tri_comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>301</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 0);
border :1px solid rgb(255, 255, 255);
border-radius :0px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
</string>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Asc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Desc</string>
        </property>
       </item>
      </widget>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>301</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </widget>
     <zorder>label_2</zorder>
     <zorder>tableView</zorder>
     <zorder>label_44</zorder>
     <zorder>label_40</zorder>
     <zorder>groupBox</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QDateTimeEdit" name="date">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>220</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>220</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 228, 228);</string>
      </property>
      <property name="text">
       <string>Date :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 228, 228);</string>
      </property>
      <property name="text">
       <string>Catégorie :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="titre">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="description">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>310</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 228, 228);</string>
      </property>
      <property name="text">
       <string>Description :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter_button">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>410</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);
border :1px solid rgb(255, 255, 255);
border-radius :10px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
</string>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 228, 228);</string>
      </property>
      <property name="text">
       <string>Titre  :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>410</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);
border :1px solid rgb(255, 255, 255);
border-radius :10px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
</string>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QComboBox" name="category">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>190</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sports</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Comedy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Drama</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Festival</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Concert</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>851</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>280</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 227, 228);</string>
      </property>
      <property name="text">
       <string>Latitude :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="longtitude">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>250</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>250</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 227, 228);</string>
      </property>
      <property name="text">
       <string>Longtitude  :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="latitude">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>280</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>date</zorder>
     <zorder>label_21</zorder>
     <zorder>label_23</zorder>
     <zorder>titre</zorder>
     <zorder>description</zorder>
     <zorder>label_22</zorder>
     <zorder>ajouter_button</zorder>
     <zorder>label_20</zorder>
     <zorder>pushButton_12</zorder>
     <zorder>category</zorder>
     <zorder>label_30</zorder>
     <zorder>longtitude</zorder>
     <zorder>label_24</zorder>
     <zorder>latitude</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QPushButton" name="Rechercher_button">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>120</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);
border :1px solid rgb(255, 255, 255);
border-radius :10px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
</string>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>310</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 228, 228);</string>
      </property>
      <property name="text">
       <string>Description :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 228, 228);</string>
      </property>
      <property name="text">
       <string>Catégorie :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>220</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 228, 228);</string>
      </property>
      <property name="text">
       <string>Date :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_modif">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>130</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier_button">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>400</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);
border :1px solid rgb(255, 255, 255);
border-radius :10px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
</string>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 228, 228);</string>
      </property>
      <property name="text">
       <string>Titre  :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="titre_modif">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 228, 228);</string>
      </property>
      <property name="text">
       <string>Id Evenement :</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="date_modif">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>220</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="description_modif">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_17">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>400</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);
border :1px solid rgb(255, 255, 255);
border-radius :10px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
</string>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QComboBox" name="category_modif">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>190</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sports</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Comedy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Drama</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Festival</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Concert</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>851</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="latitude_modif">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>280</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="longtitude_modif">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>250</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>280</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 227, 228);</string>
      </property>
      <property name="text">
       <string>Latitude :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>250</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 227, 228);</string>
      </property>
      <property name="text">
       <string>Longtitude  :</string>
      </property>
     </widget>
     <zorder>label_4</zorder>
     <zorder>Rechercher_button</zorder>
     <zorder>label_27</zorder>
     <zorder>label_28</zorder>
     <zorder>label_25</zorder>
     <zorder>id_modif</zorder>
     <zorder>modifier_button</zorder>
     <zorder>label_29</zorder>
     <zorder>titre_modif</zorder>
     <zorder>label_26</zorder>
     <zorder>date_modif</zorder>
     <zorder>description_modif</zorder>
     <zorder>pushButton_17</zorder>
     <zorder>category_modif</zorder>
     <zorder>latitude_modif</zorder>
     <zorder>longtitude_modif</zorder>
     <zorder>label_31</zorder>
     <zorder>label_32</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QPushButton" name="supprimer_button">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>340</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);
border :1px solid rgb(255, 255, 255);
border-radius :10px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_supprime">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>250</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_43">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 228, 228);</string>
      </property>
      <property name="text">
       <string>Id Evenement :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>851</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_5</zorder>
     <zorder>supprimer_button</zorder>
     <zorder>id_supprime</zorder>
     <zorder>label_43</zorder>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Statistique</string>
     </attribute>
     <widget class="QPushButton" name="Statistique_button">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>0</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_stat">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>821</width>
        <height>501</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Localisation</string>
     </attribute>
     <widget class="QLineEdit" name="id_location">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>240</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>250</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 228, 228);</string>
      </property>
      <property name="text">
       <string>Id_Evenement :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Location">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>240</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/Map.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>851</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_6</zorder>
     <zorder>id_location</zorder>
     <zorder>label</zorder>
     <zorder>Location</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
